### ISSUE TITLE
Youtube music doesn't scrobble as the site got updated

### ISSUE THREAD

**Issue by far0813** [web-scrobbler/web-scrobbler#3482]

**Comment by olexandr-perepiyaka** [comment]
Same problem. Music playing. Extension popup show "Play some music" message.
Browser: Microsoft Edge; 107.0.1418.52 (64-bit)
Extension: Web Scrobbler 2.50.2
Page: https://music.youtube.com/playlist?list=OLAK5uy_mMd5T3BP_go8z6mitQJnkcQ-l74_LUCQ8

Web page conlole log have messages about playing tracks, like:
Web Scrobbler: {
  "track": "SIMPLY THE BEST",
  "artist": "Black Eyed Peas, Anitta, El Alfa",
  "album": "ELEVATION",
  "albumArtist": null,
  "uniqueID": "QAdKInhW-Aw",
  "duration": 236,
  "currentTime": null,
  "isPlaying": false,
  "trackArt": "https://lh3.googleusercontent.com/_gJM-9U-4G_3yAmJW-a6aaGNOUn6_MBB0vfTLsUx3DWYva8LcbRDQy4ztAuGNQJQN7boPjw0-RhcIvnQ",
  "isPodcast": false,
  "originUrl": "https://music.youtube.com/playlist?list=OLAK5uy_mMd5T3BP_go8z6mitQJnkcQ-l74_LUCQ8"
}

Extension backgroung page log has message "Created controller for YouTube Music connector". But no scrobble logs.

**Comment by jaccarmac** [comment]
@far0813 Which browser are you using?

@olexandr-perepiyaka Can you duplicate the issue in Chrome or Firefox?

I checked for the regression in Chrome yesterday and didn't see it and am seeing a few Edge issues in the tracker at the moment. However, Google is also known to roll out site changes gradually.

**Comment by thexenographer** [comment]
I'm having the same problem in Opera 93.0.4585.21, for what it's worth, and I tried a number of different tracks with the same results for all of them. Though in my case I'm not seeing anything in the console log suggesting that it recognizes the track that's playing.

**Comment by DonGrobione** [comment]
Samie issue with Edge Version 107.0.1418.56 across two PCs. But it worked with Firefox 107, so I guess something in Edge changed.

**Comment by Elfanaarg** [comment]
The version on Ms Stor hasnt been updated since January, the Chrome store one was updated 7 days ago. Uninstall and download it from chrome store

**Comment by far0813** [comment]
> 

i use microsoft edge, i tried using it in chrome and it works thank you

**Comment by far0813** [comment]
@jaccarmac i use microsoft edge, i tried using chrome and it works thank you

**Comment by Elfanaarg** [comment]
You can install it on Edge from the crhome store, thats what i´ve done

**Comment by thexenographer** [comment]
Opera doesn't have its own version, so I was already using the version installed from the Chrome store and it's still not working. I also tried Chrome itself (version 107.0.5304.122) and still had the exact same problem. I've tried this on two different laptops and it's been the same on both. But if installing the addon from the Chrome store worked for everyone else, I guess somehow it's just me?
